desktop/components/my_active_bids/helpers.coffee

```diff
@@ -1,6 +1,6 @@
 moment = require 'moment'
-{ PREDICTION_URL } = require('sharify').data
+Sale = require '../../models/sale.coffee'
+{PREDICTION_URL} = require('sharify').data
+{ liveAuctionUrl } = require '../../../utils/urls.js'
 
-module.exports =
-  liveAuctionUrl: (saleId) ->
-    "#{PREDICTION_URL}/#{saleId}"
+module.exports = liveAuctionUrl
```

Comment:
Rather than locking down `helpers.coffee` to a single function `module.exports === require('helpers') === liveAuctionUrl` I would expand it as an object so that it carries the assumption that `helpers.coffee` (plural naming) can contain many helpers: 
```coffeescript
module.exports =
  liveAuctionUrl: liveAuctionUrl 
```
which later you can use like
```coffeescript
helpers = require 'helpers'
...
url = helpers.liveAuctionUrl
```
Otherwise, with some slight indirection, 
```coffeescript
liveAuctionUrl = require 'helpers' 
```